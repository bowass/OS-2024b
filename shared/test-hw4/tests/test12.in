63336
56394
57815
29014
1644
15868
25517
48744
21119
32929
7154
11890
54143
21525
44229
59117
49710
21919
28330
48851
36043
7190
26939
55570
13199
59015
41602
11950
60304
4795
62948
35511
47056
22686
50964
42430
44527
41921
8854
39827
34352
30316
10869
48887
10778
35512
9905
983
56746
56158
3287
41374
12287
49271
12604
25252
27535
24033
6421
29524
33663
31112
7380
7798
16035
17804
25220
34163
61046
65301
14376
37738
49027
22861
25877
56433
55187
46601
9768
19135
14969
62210
49176
54427
41074
44535
30583
460
58566
813
37826
5478
54787
42218
19923
56070
2917
32737
15018
38869
27783
35737
59989
31369
17653
37602
43979
57958
36460
14301
23017
37793
33647
31269
22147
11264
40537
45969
9735
18481
6871
41293
38192
64395
62662
9795
2788
45362
6093
15463
51995
11502
40586
834
7795
60860
26638
44476
41027
42627
7493
59451
52819
29125
4163
38796
17149
2882
8598
39386
43628
27732
22466
35984
44670
40283
40458
1725
52903
57693
38175
65104
36361
40101
19473
18245
21338
21238
21398
11472
30566
20902
19107
56515
34496
31577
33104
16917
9095
59358
2065
47853
15957
54376
34181
46985
23896
1572
37268
7787
56449
50157
47058
45003
3141
12508
19951
48400
16433
1127
30897
9386
40431
26709
53537
40800
26164
22901
52981
37133
31489
20549
58297
45435
48887
59476
59017
17570
43703
50390
18401
62044
30326
55896
3378
10029
9048
17890
29190
8396
14088
2440
55566
21262
48734
52333
3514
21782
53838
56321
47538
61337
36512
62065
47097
58769
40135
54208
28467
5431
56265
39850
54963
59964
8200
24649
38047
21754
16229
5442
9304
30990
53906
47642
39648
36928
26552
27027
36748
43307
34754
36822
55947
49340
25926
49521
10955
2772
63538
55733
48869
63980
12067
49966
33803
1087
16616
53415
5792
58031
7605
31419
49056
183
30625
57319
60910
39666
11509
27718
51774
16751
12973
54412
48184
34691
61237
18054
7175
35728
53472
21634
17761
20093
37214
13132
27398
21496
49097
34981
58279
33904
56344
40505
26619
9240
10168
9202
57721
63379
32207
2114
8812
9545
10181
54881
1840
30224
23625
62821
52653
51464
24043
42909
15885
25579
24635
51935
32808
5730
31363
47261
4577
970
31983
5598
57506
5563
53938
19600
35856
34280
21834
60302
49542
20576
47100
15101
22545
854
30846
17191
12627
40007
36614
54699
63957
37847
25646
57486
32728
44211
8872
59677
38428
62963
40791
38675
7080
43644
40694
50391
60953
16910
23039
33179
30000
22572
44251
9201
54292
51078
22125
12752
32291
134
38612
6769
13915
53415
7216
63088
19357
41127
34078
63584
33870
14461
44043
13925
45356
36107
15226
58451
2595
49375
30895
51811
304
38155
61186
25006
28560
64157
49320
8383
51028
17974
22379
61899
37433
48336
16509
47768
58042
27276
50943
40779
61157
64121
19140
22976
13859
64138
26914
39031
57056
3481
10612
48100
17568
28419
10224
45113
38112
52435
14922
22239
39256
22332
2651
48788
1644
59843
30697
47132
25709
11694
48128
39789
62610
2982
56448
40938
32616
28723
62790
32195
6221
13716
36961
34739
52884
43534
33513
2507
16132
58739
5284
23895