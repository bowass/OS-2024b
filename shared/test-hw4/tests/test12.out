Virtual address: 63336 Physical address: 104 Value: 0
Virtual address: 56394 Physical address: 330 Value: 55
Virtual address: 57815 Physical address: 727 Value: 117
Virtual address: 29014 Physical address: 854 Value: 28
Virtual address: 1644 Physical address: 1132 Value: 0
Virtual address: 15868 Physical address: 1532 Value: 0
Virtual address: 25517 Physical address: 1709 Value: 0
Virtual address: 48744 Physical address: 1896 Value: 0
Virtual address: 21119 Physical address: 2175 Value: -97
Virtual address: 32929 Physical address: 2465 Value: 0
Virtual address: 7154 Physical address: 2802 Value: 6
Virtual address: 11890 Physical address: 2930 Value: 11
Virtual address: 54143 Physical address: 3199 Value: -33
Virtual address: 21525 Physical address: 3349 Value: 0
Virtual address: 44229 Physical address: 3781 Value: 0
Virtual address: 59117 Physical address: 4077 Value: 0
Virtual address: 49710 Physical address: 4142 Value: 48
Virtual address: 21919 Physical address: 4511 Value: 103
Virtual address: 28330 Physical address: 4778 Value: 27
Virtual address: 48851 Physical address: 2003 Value: -76
Virtual address: 36043 Physical address: 5067 Value: 50
Virtual address: 7190 Physical address: 5142 Value: 7
Virtual address: 26939 Physical address: 5435 Value: 78
Virtual address: 55570 Physical address: 5650 Value: 54
Virtual address: 13199 Physical address: 6031 Value: -29
Virtual address: 59015 Physical address: 3975 Value: -95
Virtual address: 41602 Physical address: 6274 Value: 40
Virtual address: 11950 Physical address: 2990 Value: 11
Virtual address: 60304 Physical address: 6544 Value: 0
Virtual address: 4795 Physical address: 6843 Value: -82
Virtual address: 62948 Physical address: 7140 Value: 0
Virtual address: 35511 Physical address: 7351 Value: -83
Virtual address: 47056 Physical address: 7632 Value: 0
Virtual address: 22686 Physical address: 7838 Value: 22
Virtual address: 50964 Physical address: 7956 Value: 0
Virtual address: 42430 Physical address: 8382 Value: 41
Virtual address: 44527 Physical address: 8687 Value: 123
Virtual address: 41921 Physical address: 8897 Value: 0
Virtual address: 8854 Physical address: 9110 Value: 8
Virtual address: 39827 Physical address: 9363 Value: -28
Virtual address: 34352 Physical address: 9520 Value: 0
Virtual address: 30316 Physical address: 9836 Value: 0
Virtual address: 10869 Physical address: 10101 Value: 0
Virtual address: 48887 Physical address: 2039 Value: -67
Virtual address: 10778 Physical address: 10010 Value: 10
Virtual address: 35512 Physical address: 7352 Value: 0
Virtual address: 9905 Physical address: 10417 Value: 0
Virtual address: 983 Physical address: 10711 Value: -11
Virtual address: 56746 Physical address: 10922 Value: 55
Virtual address: 56158 Physical address: 11102 Value: 54
Virtual address: 3287 Physical address: 11479 Value: 53
Virtual address: 41374 Physical address: 11678 Value: 40
Virtual address: 12287 Physical address: 12031 Value: -1
Virtual address: 49271 Physical address: 12151 Value: 29
Virtual address: 12604 Physical address: 12348 Value: 0
Virtual address: 25252 Physical address: 12708 Value: 0
Virtual address: 27535 Physical address: 12943 Value: -29
Virtual address: 24033 Physical address: 13281 Value: 0
Virtual address: 6421 Physical address: 13333 Value: 0
Virtual address: 29524 Physical address: 13652 Value: 0
Virtual address: 33663 Physical address: 13951 Value: -33
Virtual address: 31112 Physical address: 14216 Value: 0
Virtual address: 7380 Physical address: 5332 Value: 0
Virtual address: 7798 Physical address: 14454 Value: 7
Virtual address: 16035 Physical address: 14755 Value: -88
Virtual address: 17804 Physical address: 14988 Value: 0
Virtual address: 25220 Physical address: 12676 Value: 0
Virtual address: 34163 Physical address: 15219 Value: 92
Virtual address: 61046 Physical address: 15478 Value: 59
Virtual address: 65301 Physical address: 15637 Value: 0
Virtual address: 14376 Physical address: 15912 Value: 0
Virtual address: 37738 Physical address: 16234 Value: 36
Virtual address: 49027 Physical address: 16515 Value: -32
Virtual address: 22861 Physical address: 16717 Value: 0
Virtual address: 25877 Physical address: 16917 Value: 0
Virtual address: 56433 Physical address: 369 Value: 0
Virtual address: 55187 Physical address: 17299 Value: -28
Virtual address: 46601 Physical address: 17417 Value: 0
Virtual address: 9768 Physical address: 10280 Value: 0
Virtual address: 19135 Physical address: 17855 Value: -81
Virtual address: 14969 Physical address: 18041 Value: 0
Virtual address: 62210 Physical address: 18178 Value: 60
Virtual address: 49176 Physical address: 12056 Value: 0
Virtual address: 54427 Physical address: 18587 Value: 38
Virtual address: 41074 Physical address: 18802 Value: 40
Virtual address: 44535 Physical address: 8695 Value: 125
Virtual address: 30583 Physical address: 19063 Value: -35
Virtual address: 460 Physical address: 19404 Value: 0
Virtual address: 58566 Physical address: 19654 Value: 57
Virtual address: 813 Physical address: 10541 Value: 0
Virtual address: 37826 Physical address: 16322 Value: 36
Virtual address: 5478 Physical address: 19814 Value: 5
Virtual address: 54787 Physical address: 19971 Value: -128
Virtual address: 42218 Physical address: 20458 Value: 41
Virtual address: 19923 Physical address: 20691 Value: 116
Virtual address: 56070 Physical address: 11014 Value: 54
Virtual address: 2917 Physical address: 20837 Value: 0
Virtual address: 32737 Physical address: 21217 Value: 0
Virtual address: 15018 Physical address: 18090 Value: 14
Virtual address: 38869 Physical address: 21461 Value: 0
Virtual address: 27783 Physical address: 21639 Value: 33
Virtual address: 35737 Physical address: 21913 Value: 0
Virtual address: 59989 Physical address: 22101 Value: 0
Virtual address: 31369 Physical address: 22409 Value: 0
Virtual address: 17653 Physical address: 22773 Value: 0
Virtual address: 37602 Physical address: 23010 Value: 36
Virtual address: 43979 Physical address: 23243 Value: -14
Virtual address: 57958 Physical address: 23398 Value: 56
Virtual address: 36460 Physical address: 23660 Value: 0
Virtual address: 14301 Physical address: 24029 Value: 0
Virtual address: 23017 Physical address: 16873 Value: 0
Virtual address: 37793 Physical address: 16289 Value: 0
Virtual address: 33647 Physical address: 13935 Value: -37
Virtual address: 31269 Physical address: 22309 Value: 0
Virtual address: 22147 Physical address: 24195 Value: -96
Virtual address: 11264 Physical address: 24320 Value: 0
Virtual address: 40537 Physical address: 24665 Value: 0
Virtual address: 45969 Physical address: 24977 Value: 0
Virtual address: 9735 Physical address: 10247 Value: -127
Virtual address: 18481 Physical address: 25137 Value: 0
Virtual address: 6871 Physical address: 25559 Value: -75
Virtual address: 41293 Physical address: 11597 Value: 0
Virtual address: 38192 Physical address: 25648 Value: 0
Virtual address: 64395 Physical address: 25995 Value: -30
Virtual address: 62662 Physical address: 26310 Value: 61
Virtual address: 9795 Physical address: 10307 Value: -112
Virtual address: 2788 Physical address: 26596 Value: 0
Virtual address: 45362 Physical address: 26674 Value: 44
Virtual address: 6093 Physical address: 27085 Value: 0
Virtual address: 15463 Physical address: 27239 Value: 25
Virtual address: 51995 Physical address: 27419 Value: -58
Virtual address: 11502 Physical address: 24558 Value: 11
Virtual address: 40586 Physical address: 24714 Value: 39
Virtual address: 834 Physical address: 10562 Value: 0
Virtual address: 7795 Physical address: 14451 Value: -100
Virtual address: 60860 Physical address: 27836 Value: 0
Virtual address: 26638 Physical address: 27918 Value: 26
Virtual address: 44476 Physical address: 8636 Value: 0
Virtual address: 41027 Physical address: 18755 Value: 16
Virtual address: 42627 Physical address: 28291 Value: -96
Virtual address: 7493 Physical address: 28485 Value: 0
Virtual address: 59451 Physical address: 28731 Value: 14
Virtual address: 52819 Physical address: 29011 Value: -108
Virtual address: 29125 Physical address: 965 Value: 0
Virtual address: 4163 Physical address: 29251 Value: 16
Virtual address: 38796 Physical address: 21388 Value: 0
Virtual address: 17149 Physical address: 29693 Value: 0
Virtual address: 2882 Physical address: 20802 Value: 2
Virtual address: 8598 Physical address: 29846 Value: 8
Virtual address: 39386 Physical address: 30170 Value: 38
Virtual address: 43628 Physical address: 30316 Value: 0
Virtual address: 27732 Physical address: 21588 Value: 0
Virtual address: 22466 Physical address: 30658 Value: 21
Virtual address: 35984 Physical address: 5008 Value: 0
Virtual address: 44670 Physical address: 30846 Value: 43
Virtual address: 40283 Physical address: 31067 Value: 86
Virtual address: 40458 Physical address: 24586 Value: 39
Virtual address: 1725 Physical address: 1213 Value: 0
Virtual address: 52903 Physical address: 29095 Value: -87
Virtual address: 57693 Physical address: 605 Value: 0
Virtual address: 38175 Physical address: 25631 Value: 71
Virtual address: 65104 Physical address: 31312 Value: 0
Virtual address: 36361 Physical address: 23561 Value: 0
Virtual address: 40101 Physical address: 31653 Value: 0
Virtual address: 19473 Physical address: 31761 Value: 0
Virtual address: 18245 Physical address: 32069 Value: 0
Virtual address: 21338 Physical address: 32346 Value: 20
Virtual address: 21238 Physical address: 2294 Value: 20
Virtual address: 21398 Physical address: 32406 Value: 20
Virtual address: 11472 Physical address: 24528 Value: 0
Virtual address: 30566 Physical address: 19046 Value: 29
Virtual address: 20902 Physical address: 32678 Value: 20
Virtual address: 19107 Physical address: 17827 Value: -88
Virtual address: 56515 Physical address: 451 Value: 48
Virtual address: 34496 Physical address: 9664 Value: 0
Virtual address: 31577 Physical address: 32857 Value: 0
Virtual address: 33104 Physical address: 33104 Value: 0
Virtual address: 16917 Physical address: 29461 Value: 0
Virtual address: 9095 Physical address: 33415 Value: -31
Virtual address: 59358 Physical address: 33758 Value: 57
Virtual address: 2065 Physical address: 33809 Value: 0
Virtual address: 47853 Physical address: 34285 Value: 0
Virtual address: 15957 Physical address: 14677 Value: 0
Virtual address: 54376 Physical address: 18536 Value: 0
Virtual address: 34181 Physical address: 15237 Value: 0
Virtual address: 46985 Physical address: 7561 Value: 0
Virtual address: 23896 Physical address: 13144 Value: 0
Virtual address: 1572 Physical address: 1060 Value: 0
Virtual address: 37268 Physical address: 34452 Value: 0
Virtual address: 7787 Physical address: 14443 Value: -102
Virtual address: 56449 Physical address: 385 Value: 0
Virtual address: 50157 Physical address: 34797 Value: 0
Virtual address: 47058 Physical address: 7634 Value: 45
Virtual address: 45003 Physical address: 35019 Value: -14
Virtual address: 3141 Physical address: 11333 Value: 0
Virtual address: 12508 Physical address: 35292 Value: 0
Virtual address: 19951 Physical address: 20719 Value: 123
Virtual address: 48400 Physical address: 35344 Value: 0
Virtual address: 16433 Physical address: 35633 Value: 0
Virtual address: 1127 Physical address: 35943 Value: 25
Virtual address: 30897 Physical address: 36273 Value: 0
Virtual address: 9386 Physical address: 36522 Value: 9
Virtual address: 40431 Physical address: 31215 Value: 123
Virtual address: 26709 Physical address: 27989 Value: 0
Virtual address: 53537 Physical address: 36641 Value: 0
Virtual address: 40800 Physical address: 36960 Value: 0
Virtual address: 26164 Physical address: 37172 Value: 0
Virtual address: 22901 Physical address: 16757 Value: 0
Virtual address: 52981 Physical address: 29173 Value: 0
Virtual address: 37133 Physical address: 34317 Value: 0
Virtual address: 31489 Physical address: 32769 Value: 0
Virtual address: 20549 Physical address: 37445 Value: 0
Virtual address: 58297 Physical address: 37817 Value: 0
Virtual address: 45435 Physical address: 26747 Value: 94
Virtual address: 48887 Physical address: 2039 Value: -67
Virtual address: 59476 Physical address: 28756 Value: 0
Virtual address: 59017 Physical address: 3977 Value: 0
Virtual address: 17570 Physical address: 22690 Value: 17
Virtual address: 43703 Physical address: 30391 Value: -83
Virtual address: 50390 Physical address: 38102 Value: 49
Virtual address: 18401 Physical address: 32225 Value: 0
Virtual address: 62044 Physical address: 38236 Value: 0
Virtual address: 30326 Physical address: 9846 Value: 29
Virtual address: 55896 Physical address: 38488 Value: 0
Virtual address: 3378 Physical address: 38706 Value: 3
Virtual address: 10029 Physical address: 38957 Value: 0
Virtual address: 9048 Physical address: 33368 Value: 0
Virtual address: 17890 Physical address: 15074 Value: 17
Virtual address: 29190 Physical address: 39174 Value: 28
Virtual address: 8396 Physical address: 39628 Value: 0
Virtual address: 14088 Physical address: 23816 Value: 0
Virtual address: 2440 Physical address: 39816 Value: 0
Virtual address: 55566 Physical address: 5646 Value: 54
Virtual address: 21262 Physical address: 32270 Value: 20
Virtual address: 48734 Physical address: 1886 Value: 47
Virtual address: 52333 Physical address: 40045 Value: 0
Virtual address: 3514 Physical address: 38842 Value: 3
Virtual address: 21782 Physical address: 4374 Value: 21
Virtual address: 53838 Physical address: 40270 Value: 52
Virtual address: 56321 Physical address: 257 Value: 0
Virtual address: 47538 Physical address: 40626 Value: 46
Virtual address: 61337 Physical address: 40857 Value: 0
Virtual address: 36512 Physical address: 23712 Value: 0
Virtual address: 62065 Physical address: 38257 Value: 0
Virtual address: 47097 Physical address: 7673 Value: 0
Virtual address: 58769 Physical address: 41105 Value: 0
Virtual address: 40135 Physical address: 31687 Value: 49
Virtual address: 54208 Physical address: 3264 Value: 0
Virtual address: 28467 Physical address: 41267 Value: -52
Virtual address: 5431 Physical address: 19767 Value: 77
Virtual address: 56265 Physical address: 11209 Value: 0
Virtual address: 39850 Physical address: 9386 Value: 38
Virtual address: 54963 Physical address: 20147 Value: -84
Virtual address: 59964 Physical address: 22076 Value: 0
Virtual address: 8200 Physical address: 39432 Value: 0
Virtual address: 24649 Physical address: 41545 Value: 0
Virtual address: 38047 Physical address: 41887 Value: 39
Virtual address: 21754 Physical address: 3578 Value: 21
Virtual address: 16229 Physical address: 42085 Value: 0
Virtual address: 5442 Physical address: 19778 Value: 5
Virtual address: 9304 Physical address: 36440 Value: 0
Virtual address: 30990 Physical address: 14094 Value: 30
Virtual address: 53906 Physical address: 40338 Value: 52
Virtual address: 47642 Physical address: 34074 Value: 46
Virtual address: 39648 Physical address: 42464 Value: 0
Virtual address: 36928 Physical address: 42560 Value: 0
Virtual address: 26552 Physical address: 42936 Value: 0
Virtual address: 27027 Physical address: 5523 Value: 100
Virtual address: 36748 Physical address: 43148 Value: 0
Virtual address: 43307 Physical address: 43307 Value: 74
Virtual address: 34754 Physical address: 43714 Value: 33
Virtual address: 36822 Physical address: 43222 Value: 35
Virtual address: 55947 Physical address: 38539 Value: -94
Virtual address: 49340 Physical address: 12220 Value: 0
Virtual address: 25926 Physical address: 16966 Value: 25
Virtual address: 49521 Physical address: 43889 Value: 0
Virtual address: 10955 Physical address: 10187 Value: -78
Virtual address: 2772 Physical address: 26580 Value: 0
Virtual address: 63538 Physical address: 44082 Value: 62
Virtual address: 55733 Physical address: 5813 Value: 0
Virtual address: 48869 Physical address: 2021 Value: 0
Virtual address: 63980 Physical address: 44524 Value: 0
Virtual address: 12067 Physical address: 11811 Value: -56
Virtual address: 49966 Physical address: 34606 Value: 48
Virtual address: 33803 Physical address: 44555 Value: 2
Virtual address: 1087 Physical address: 35903 Value: 15
Virtual address: 16616 Physical address: 35816 Value: 0
Virtual address: 53415 Physical address: 44967 Value: 41
Virtual address: 5792 Physical address: 45216 Value: 0
Virtual address: 58031 Physical address: 23471 Value: -85
Virtual address: 7605 Physical address: 28597 Value: 0
Virtual address: 31419 Physical address: 22459 Value: -82
Virtual address: 49056 Physical address: 16544 Value: 0
Virtual address: 183 Physical address: 45495 Value: 45
Virtual address: 30625 Physical address: 19105 Value: 0
Virtual address: 57319 Physical address: 45799 Value: -7
Virtual address: 60910 Physical address: 27886 Value: 59
Virtual address: 39666 Physical address: 42482 Value: 38
Virtual address: 11509 Physical address: 24565 Value: 0
Virtual address: 27718 Physical address: 21574 Value: 27
Virtual address: 51774 Physical address: 45886 Value: 50
Virtual address: 16751 Physical address: 46191 Value: 91
Virtual address: 12973 Physical address: 46509 Value: 0
Virtual address: 54412 Physical address: 18572 Value: 0
Virtual address: 48184 Physical address: 46648 Value: 0
Virtual address: 34691 Physical address: 43651 Value: -32
Virtual address: 61237 Physical address: 40757 Value: 0
Virtual address: 18054 Physical address: 46982 Value: 17
Virtual address: 7175 Physical address: 5127 Value: 1
Virtual address: 35728 Physical address: 21904 Value: 0
Virtual address: 53472 Physical address: 45024 Value: 0
Virtual address: 21634 Physical address: 3458 Value: 21
Virtual address: 17761 Physical address: 14945 Value: 0
Virtual address: 20093 Physical address: 47229 Value: 0
Virtual address: 37214 Physical address: 34398 Value: 36
Virtual address: 13132 Physical address: 5964 Value: 0
Virtual address: 27398 Physical address: 12806 Value: 26
Virtual address: 21496 Physical address: 32504 Value: 0
Virtual address: 49097 Physical address: 16585 Value: 0
Virtual address: 34981 Physical address: 47525 Value: 0
Virtual address: 58279 Physical address: 37799 Value: -23
Virtual address: 33904 Physical address: 44656 Value: 0
Virtual address: 56344 Physical address: 280 Value: 0
Virtual address: 40505 Physical address: 24633 Value: 0
Virtual address: 26619 Physical address: 43003 Value: -2
Virtual address: 9240 Physical address: 36376 Value: 0
Virtual address: 10168 Physical address: 39096 Value: 0
Virtual address: 9202 Physical address: 33522 Value: 8
Virtual address: 57721 Physical address: 633 Value: 0
Virtual address: 63379 Physical address: 147 Value: -28
Virtual address: 32207 Physical address: 47823 Value: 115
Virtual address: 2114 Physical address: 33858 Value: 2
Virtual address: 8812 Physical address: 9068 Value: 0
Virtual address: 9545 Physical address: 47945 Value: 0
Virtual address: 10181 Physical address: 39109 Value: 0
Virtual address: 54881 Physical address: 20065 Value: 0
Virtual address: 1840 Physical address: 48176 Value: 0
Virtual address: 30224 Physical address: 9744 Value: 0
Virtual address: 23625 Physical address: 48457 Value: 0
Virtual address: 62821 Physical address: 7013 Value: 0
Virtual address: 52653 Physical address: 48813 Value: 0
Virtual address: 51464 Physical address: 48904 Value: 0
Virtual address: 24043 Physical address: 13291 Value: 122
Virtual address: 42909 Physical address: 49309 Value: 0
Virtual address: 15885 Physical address: 14605 Value: 0
Virtual address: 25579 Physical address: 1771 Value: -6
Virtual address: 24635 Physical address: 41531 Value: 14
Virtual address: 51935 Physical address: 46047 Value: -73
Virtual address: 32808 Physical address: 2344 Value: 0
Virtual address: 5730 Physical address: 45154 Value: 5
Virtual address: 31363 Physical address: 22403 Value: -96
Virtual address: 47261 Physical address: 49565 Value: 0
Virtual address: 4577 Physical address: 49889 Value: 0
Virtual address: 970 Physical address: 10698 Value: 0
Virtual address: 31983 Physical address: 50159 Value: 59
Virtual address: 5598 Physical address: 19934 Value: 5
Virtual address: 57506 Physical address: 50338 Value: 56
Virtual address: 5563 Physical address: 19899 Value: 110
Virtual address: 53938 Physical address: 40370 Value: 52
Virtual address: 19600 Physical address: 31888 Value: 0
Virtual address: 35856 Physical address: 4880 Value: 0
Virtual address: 34280 Physical address: 15336 Value: 0
Virtual address: 21834 Physical address: 4426 Value: 21
Virtual address: 60302 Physical address: 6542 Value: 58
Virtual address: 49542 Physical address: 43910 Value: 48
Virtual address: 20576 Physical address: 37472 Value: 0
Virtual address: 47100 Physical address: 7676 Value: 0
Virtual address: 15101 Physical address: 18173 Value: 0
Virtual address: 22545 Physical address: 7697 Value: 0
Virtual address: 854 Physical address: 10582 Value: 0
Virtual address: 30846 Physical address: 36222 Value: 30
Virtual address: 17191 Physical address: 50471 Value: -55
Virtual address: 12627 Physical address: 12371 Value: 84
Virtual address: 40007 Physical address: 31559 Value: 17
Virtual address: 36614 Physical address: 43014 Value: 35
Virtual address: 54699 Physical address: 50859 Value: 106
Virtual address: 63957 Physical address: 44501 Value: 0
Virtual address: 37847 Physical address: 16343 Value: -11
Virtual address: 25646 Physical address: 50990 Value: 25
Virtual address: 57486 Physical address: 50318 Value: 56
Virtual address: 32728 Physical address: 21208 Value: 0
Virtual address: 44211 Physical address: 3763 Value: 44
Virtual address: 8872 Physical address: 9128 Value: 0
Virtual address: 59677 Physical address: 51229 Value: 0
Virtual address: 38428 Physical address: 51484 Value: 0
Virtual address: 62963 Physical address: 7155 Value: 124
Virtual address: 40791 Physical address: 36951 Value: -43
Virtual address: 38675 Physical address: 21267 Value: -60
Virtual address: 7080 Physical address: 2728 Value: 0
Virtual address: 43644 Physical address: 30332 Value: 0
Virtual address: 40694 Physical address: 24822 Value: 39
Virtual address: 50391 Physical address: 38103 Value: 53
Virtual address: 60953 Physical address: 15385 Value: 0
Virtual address: 16910 Physical address: 29454 Value: 16
Virtual address: 23039 Physical address: 16895 Value: 127
Virtual address: 33179 Physical address: 33179 Value: 102
Virtual address: 30000 Physical address: 51760 Value: 0
Virtual address: 22572 Physical address: 7724 Value: 0
Virtual address: 44251 Physical address: 3803 Value: 54
Virtual address: 9201 Physical address: 33521 Value: 0
Virtual address: 54292 Physical address: 18452 Value: 0
Virtual address: 51078 Physical address: 8070 Value: 49
Virtual address: 22125 Physical address: 24173 Value: 0
Virtual address: 12752 Physical address: 12496 Value: 0
Virtual address: 32291 Physical address: 52003 Value: -120
Virtual address: 134 Physical address: 45446 Value: 0
Virtual address: 38612 Physical address: 51668 Value: 0
Virtual address: 6769 Physical address: 25457 Value: 0
Virtual address: 13915 Physical address: 52315 Value: -106
Virtual address: 53415 Physical address: 44967 Value: 41
Virtual address: 7216 Physical address: 5168 Value: 0
Virtual address: 63088 Physical address: 52592 Value: 0
Virtual address: 19357 Physical address: 52893 Value: 0
Virtual address: 41127 Physical address: 18855 Value: 41
Virtual address: 34078 Physical address: 15134 Value: 33
Virtual address: 63584 Physical address: 44128 Value: 0
Virtual address: 33870 Physical address: 44622 Value: 33
Virtual address: 14461 Physical address: 15997 Value: 0
Virtual address: 44043 Physical address: 3595 Value: 2
Virtual address: 13925 Physical address: 52325 Value: 0
Virtual address: 45356 Physical address: 26668 Value: 0
Virtual address: 36107 Physical address: 53003 Value: 66
Virtual address: 15226 Physical address: 53370 Value: 14
Virtual address: 58451 Physical address: 19539 Value: 20
Virtual address: 2595 Physical address: 26403 Value: -120
Virtual address: 49375 Physical address: 12255 Value: 55
Virtual address: 30895 Physical address: 36271 Value: 43
Virtual address: 51811 Physical address: 45923 Value: -104
Virtual address: 304 Physical address: 19248 Value: 0
Virtual address: 38155 Physical address: 25611 Value: 66
Virtual address: 61186 Physical address: 40706 Value: 59
Virtual address: 25006 Physical address: 53678 Value: 24
Virtual address: 28560 Physical address: 41360 Value: 0
Virtual address: 64157 Physical address: 53917 Value: 0
Virtual address: 49320 Physical address: 12200 Value: 0
Virtual address: 8383 Physical address: 39615 Value: 47
Virtual address: 51028 Physical address: 8020 Value: 0
Virtual address: 17974 Physical address: 46902 Value: 17
Virtual address: 22379 Physical address: 30571 Value: -38
Virtual address: 61899 Physical address: 54219 Value: 114
Virtual address: 37433 Physical address: 22841 Value: 0
Virtual address: 48336 Physical address: 46800 Value: 0
Virtual address: 16509 Physical address: 35709 Value: 0
Virtual address: 47768 Physical address: 34200 Value: 0
Virtual address: 58042 Physical address: 23482 Value: 56
Virtual address: 27276 Physical address: 54412 Value: 0
Virtual address: 50943 Physical address: 54783 Value: -65
Virtual address: 40779 Physical address: 36939 Value: -46
Virtual address: 61157 Physical address: 15589 Value: 0
Virtual address: 64121 Physical address: 53881 Value: 0
Virtual address: 19140 Physical address: 17860 Value: 0
Virtual address: 22976 Physical address: 16832 Value: 0
Virtual address: 13859 Physical address: 52259 Value: -120
Virtual address: 64138 Physical address: 53898 Value: 62
Virtual address: 26914 Physical address: 5410 Value: 26
Virtual address: 39031 Physical address: 54903 Value: 29
Virtual address: 57056 Physical address: 55264 Value: 0
Virtual address: 3481 Physical address: 38809 Value: 0
Virtual address: 10612 Physical address: 55412 Value: 0
Virtual address: 48100 Physical address: 55780 Value: 0
Virtual address: 17568 Physical address: 22688 Value: 0
Virtual address: 28419 Physical address: 41219 Value: -64
Virtual address: 10224 Physical address: 39152 Value: 0
Virtual address: 45113 Physical address: 55865 Value: 0
Virtual address: 38112 Physical address: 41952 Value: 0
Virtual address: 52435 Physical address: 40147 Value: 52
Virtual address: 14922 Physical address: 17994 Value: 14
Virtual address: 22239 Physical address: 24287 Value: -73
Virtual address: 39256 Physical address: 30040 Value: 0
Virtual address: 22332 Physical address: 30524 Value: 0
Virtual address: 2651 Physical address: 26459 Value: -106
Virtual address: 48788 Physical address: 1940 Value: 0
Virtual address: 1644 Physical address: 1132 Value: 0
Virtual address: 59843 Physical address: 51395 Value: 112
Virtual address: 30697 Physical address: 19177 Value: 0
Virtual address: 47132 Physical address: 49436 Value: 0
Virtual address: 25709 Physical address: 51053 Value: 0
Virtual address: 11694 Physical address: 56238 Value: 11
Virtual address: 48128 Physical address: 46592 Value: 0
Virtual address: 39789 Physical address: 9325 Value: 0
Virtual address: 62610 Physical address: 26258 Value: 61
Virtual address: 2982 Physical address: 20902 Value: 2
Virtual address: 56448 Physical address: 384 Value: 0
Virtual address: 40938 Physical address: 37098 Value: 39
Virtual address: 32616 Physical address: 21096 Value: 0
Virtual address: 28723 Physical address: 56371 Value: 12
Virtual address: 62790 Physical address: 6982 Value: 61
Virtual address: 32195 Physical address: 47811 Value: 112
Virtual address: 6221 Physical address: 56653 Value: 0
Virtual address: 13716 Physical address: 56980 Value: 0
Virtual address: 36961 Physical address: 42593 Value: 0
Virtual address: 34739 Physical address: 43699 Value: -20
Virtual address: 52884 Physical address: 29076 Value: 0
Virtual address: 43534 Physical address: 30222 Value: 42
Virtual address: 33513 Physical address: 57321 Value: 0
Virtual address: 2507 Physical address: 39883 Value: 114
Virtual address: 16132 Physical address: 41988 Value: 0
Virtual address: 58739 Physical address: 41075 Value: 92
Virtual address: 5284 Physical address: 57508 Value: 0
Virtual address: 23895 Physical address: 13143 Value: 85
Number of Translated Addresses = 500
Page Faults = 225
Page Fault Rate = 0.450
TLB Hits = 22
TLB Hit Rate = 0.044
